### PR TITLE
README.md: switch to https for IAIS maven repository and reflect chan…

### PR DESCRIPTION
**Pull request by bimbusch**
…ged publishing policy and location of artifacts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ There are following top-level files and directories:
 
 ## Related resources
 
-Additional resources are provided for the members of the International Data Spaces Association:
+Additional resources are now provided for everyone (formerly restricted to International Data Spaces Association members):
```
Comment (sebbader) [inline README.md]:
```suggestion
Additional resources are provided:
```

File: README.md [hunk 2]
```diff
@@ -71,9 +71,8 @@ For automated validation of the Information Model POJOs (via the builder classes
 </dependency>
 ```
 
-At the moment these resources are accessible for IDSA members only. You can obtain the access credentials from this [Jive Page](https://industrialdataspace.jiveon.com/docs/DOC-1366).
-
-- UML-based visualization of the Information Model, its classes, properties and relations, including their cardinalities. It is available on the Maven Repository mentioned above, under the same base path (visualization component).
+- UML-based visualization of the Information Model, its classes, properties and relations, including their cardinalities. It is available in the Nightly and Release Packages on [GitHub](https://github.com/IndustrialDataSpace/InformationModel/releases).
```
Comment (sebbader) [inline README.md]:
```suggestion
- UML-based visualization of the Information Model, its classes, properties and relations, including their cardinalities. It is available as part of the Nightly and Release Packages on [GitHub](https://github.com/IndustrialDataSpace/InformationModel/releases).
```

File: README.md [hunk 3]
```diff
@@ -71,9 +71,8 @@ For automated validation of the Information Model POJOs (via the builder classes
...
 
+For members of the International Data Spaces Association only:
 - Working [document](https://industrialdataspace.jiveon.com/docs/DOC-1817), on the association's Jive platform, explaining
```
Comment (sebbader) [inline README.md]:
```suggestion
- Working [document](https://industrialdataspace.jiveon.com/docs/DOC-1817), explaining
```

File: README.md [hunk 4]
```diff
@@ -71,9 +71,8 @@ For automated validation of the Information Model POJOs (via the builder classes
...
 the usage of the Information Model for communication on the Industrial Data Space.
```
Comment (sebbader) [inline README.md]:
```suggestion
the usage of the Information Model for data exchanges in the IDS.```

**Comment by bimbusch** [comment]
Thanks @sebbader !

**Pull request merged**